src/Search/Type/Base.php

```diff
@@ -1,9 +1,9 @@
 <?php
-namespace Search\Type;
+namespace Search\Seach\Type;
 
 use Cake\ORM\Query;
 use Cake\Core\InstanceConfigTrait;
-use Search\Manager;
+use Search\Search\Manager;
```

Comment:
lets use `FOC\Search` then

like https://github.com/FriendsOfCake/Authenticate/ does for cake3 :)